vm/src/vm.rs

```diff
@@ -1011,7 +1021,8 @@ impl VirtualMachine {
             return self.abstract_isinstance(obj, cls);
         }
 
-        if let Ok(tuple) = PyTupleRef::try_from_object(self, cls.clone()) {
+        if cls.class().slots.flags.has_feature(PyTpFlags::TUPLE_SUBCLASS) {
+            let tuple = PyTupleRef::try_from_object(self, cls.clone()).unwrap();
```

Comment:
is this related with the change?

Reply:
No, it is not. it was a thing should be resolved by 10eb3d0.